deployment-bundle-steps.md

```diff
@@ -0,0 +1,135 @@
+# How to setup a project with deployment to S3
+
+1. Create S3 bucket for production / staging (your-bucket-name-production & your-bucket-name-staging). (If using cloudfront, see [optional settings](#optional-settings))
+
+2. Log into [circleci.com](https://circleci.com/) and add your project. Set up the following credentials: (see other members of the team for these values).
+  - Environment Variables
+    - GHUSER
+    - GHPASS
```

Comment:
It is a token, it was just named wrong up front... we can absolutely add the ability to use `GHTOKEN` and fallback to `GHPASS` if needed, that we we can move across.
This variable is consumed by deployment-helpers, so we'd need to adjust that and then update the std here. 
We should absolutely merge these back into one file and one process, shame we ended up with two :/